Admin/CrossTenantMailboxMigrationValidation.ps1

```diff
@@ -1019,36 +1065,39 @@ if ($CheckOrgs -and !$SourceIsOffline) {
 if ($SDP) {
     LoggingOn
     ConnectToEXOTenants
+    ConnectToSourceTenantAAD
+    $SourceTenantId = (Get-MgOrganization).id
+    $SourceAADApp = Get-MgServicePrincipal -All
+    Disconnect-MgGraph | Out-Null
     ConnectToTargetTenantAAD
-    $SourceTenantId = Read-Host "Please specify the SOURCE TenantId"
-    $TargetTenantId = (Get-AzureADTenantDetail).ObjectId
+    $TargetTenantId = (Get-MgOrganization).id
     CollectDataForSDP
+    Disconnect-MgGraph | Out-Null
```

Comment:
Same here

Reply:
yup, that's because I connect to AAD against the source tenant first, and then against the target tenant, with their proper disconnect to kill the MgGraph sessions cleanly. (lines 1047 and 1067)